Packs/RSANetWitness_v11_1/Integrations/RSANetWitnessv115/RSANetWitnessv115_test.py

```diff
@@ -347,3 +350,302 @@ def test_paging_command(mocker):
+    mocker.patch.object(RSANetWitnessv115, "paging_command", return_value=({}, [{"id": "INC-1", "lastUpdated": 1694188116}]))
+    mocker.patch.object(RSANetWitnessv115, "clean_old_inc_context", return_value=False)
+    mocker.patch.object(RSANetWitnessv115, "get_integration_context", return_value={})
+
+    res = get_modified_remote_data_command(client, {}, {"max_fetch": 2, "max_alerts": 2, "max_mirror_time": 0})
+    assert res.modified_incident_ids == expected_result
+
+
+def test_get_modified_remote_data_command_from_alerts(mocker):
+    """
+        Given:
+        - client with fetch parameters
+        - args with incident attributes
+        - params
+
+        When:
+            running get_modified_remote_data_command.
+
+        Then:
+            check the updated incidents list.
+    """
+    paging_data = ['INC-1']
+
+    class GetModifiedRemoteDataArgsResponse:
+        def __init__(self) -> dict:
+            self.last_update = "0"
+
+    mocker.patch.object(RSANetWitnessv115, "GetModifiedRemoteDataArgs", return_value=GetModifiedRemoteDataArgsResponse())
+    mocker.patch.object(client, "get_incident_request",
+                        return_value={"id": "INC-1", "status": 1, "alertCount": 1, "eventCount": 1})
+    mocker.patch.object(RSANetWitnessv115, "paging_command",
+                        return_value=({}, [{"id": "INC-1", "lastUpdated": 0, "alertCount": 10, "eventCount": 10}]))
+    mocker.patch.object(RSANetWitnessv115, "clean_old_inc_context", return_value=False)
+    mocker.patch.object(RSANetWitnessv115, "get_integration_context",
+                        return_value={"IncidentsDataCount": {"INC-1": {"alertCount": 1, "eventCount": 1}}})
+
+    res = get_modified_remote_data_command(client, {}, {"max_fetch": 2, "max_alerts": 2, "max_mirror_time": 0})
+    assert res.modified_incident_ids == paging_data
+
+
+def test_struct_inc_context():
+    """
+        Given:
+        - Alert count
+        - Event Count
+        - Created boolean
+
+        When:
+            running get_remote_data_command.
+
+        Then:
+            Create base incident structure.
+    """
+    expected_result = {"alertCount": 1, "eventCount": 1, "Created": True}
+    assert struct_inc_context(1, 1, True) == expected_result
+
+
+def test_clean_old_inc_context_with_non_expired_incident(mocker):
+    """
+        Given:
+        - Max mirror time aggregration (int for days)
+
+        When:
+            running get_modified_remote_data_command.
+
+        Then:
+            Clean old incident.
+    """
+    from datetime import datetime, timedelta
+    max_time_mirror_inc = 24
+    DATE_FORMAT = '%Y-%m-%dT%H:%M:%S.%MZ'
+    created_date = datetime.now() - timedelta(days=max_time_mirror_inc - 1)
+
+    mocker.patch.object(demisto, "getIntegrationContext",
+                        return_value={"IncidentsDataCount": {"INC-1": {"Created": created_date.strftime(DATE_FORMAT)}}})
+
+    assert clean_old_inc_context(max_time_mirror_inc) is None
```

Comment:
Please assert that demisto.setIntegrationContext was called with the wanted incident using assert_called_with as I [linked](https://github.com/demisto/content/blob/813c7a01bf5b19825088d14736066d46b60e85a6/Packs/AWS-SecurityHub/Integrations/AWS_SecurityHub/AWS_SecurityHub_test.py#L556) and asked before.

Reply:
it's done :)